suite/tests/common/decode.c

```diff
@@ -363,7 +363,6 @@ GLOBAL_LABEL(FUNCNAME:)
         RAW(41) RAW(0f) RAW(12) RAW(f4) /* i#319: movlhps %xmm12, xmm6 */
         RAW(41) RAW(0f) RAW(16) RAW(f4) /* i#319: movhlps %xmm12, xmm6 */
 #endif
-        mov  eax, 0
         mov  ecx, 0
         mov  edx, 0
         RAW(0f) RAW(01) RAW(c8) /* monitor */
```

Comment:
Please augment the comment so it's clear this references %rax and thus relies on ARG1 being there.

Reply:
Done.